tests/test_fields.py

```diff
@@ -247,6 +247,37 @@ class MySchema(Schema):
 
 class TestListNested:
 
+    @pytest.mark.parametrize('param', ('only', 'exclude', 'dump_only', 'load_only'))
+    def test_list_nested_only_exclude_dump_only_load_only_propagated_to_nested(self, param):
+
+        class Child(Schema):
+            name = fields.String()
+            age = fields.Integer()
+
+        class Family(Schema):
+            children = fields.List(fields.Nested(Child))
+
+        schema = Family(**{param: ['children.name']})
+        assert getattr(schema.fields['children'].container.schema, param) == {'name'}
+
+    @pytest.mark.parametrize('param', ('only', 'exclude'))
+    def test_list_nested_only_and_exclude_merged_with_nested(self, param):
+
+        class Child(Schema):
+            name = fields.String()
+            surname = fields.String()
+            age = fields.Integer()
+
+        class Family(Schema):
+            children = fields.List(fields.Nested(Child, **{param: ('name', 'surname')}))
+
+        schema = Family(**{param: ['children.name', 'children.age']})
+        expected = {
```

Comment:
Nit: It might be a bit cleaner to pass the expected value as a parameter.

```python
@pytest.mark.parametrize(
    ('param', 'expected'), [('only', {'name'}), ('exclude', {'name', 'surname', 'age'})]
)
def test...(self, param, expected):
```